layers/+emacs/org/packages.el

```diff
@@ -563,6 +568,8 @@ Headline^^            Visit entry^^               Filter^^                    Da
         ("." org-agenda-goto-today)
         ("gd" org-agenda-goto-date)))
     :config
+    (when org-enable-org-contacts-support
```

Comment:
I think that the proper way to do this is to put it in the package list at the top of the file with a `:toggle`.